common/build/eslint-config-fluid/strict.js

```diff
@@ -5,12 +5,33 @@
 
 module.exports = {
     extends: ["./recommended.js"],
+    rules: {
+        /**
+         * Require jsdoc/tsdoc comments on public/exported API items.
+         */
+        "jsdoc/require-jsdoc": [
+            "error",
+            {
+                // Indicates that only module exports should be flagged for lacking jsdoc comments
+                publicOnly: true,
+                // Prevents eslint from adding empty comment blocks when run with `--fix`
+                enableFixer: false,
+                require: {
+                    ClassDeclaration: true,
+                    FunctionDeclaration: true,
+
+                    // Will report for *any* methods on exported classes, regardless of whether or not they are public
+                    MethodDefinition: false,
+                },
+                contexts: ["TSEnumDeclaration", "TSInterfaceDeclaration", "TSTypeAliasDeclaration"],
```

Comment:
I'm not familiar with eslint configuration options - is this saying to run require-jsdoc on these TS constructs (does that work)?

Reply:
v cool, that's what I was guessing -- I didn't expect they would "just work" with typescript but that's cool :)